dev-infrastructure/templates/svc-cluster.bicep

```diff
@@ -614,13 +614,25 @@ module fpaCertificate '../modules/keyvault/key-vault-cert.bicep' = if (manageFpa
 
 module svcNSP '../modules/network/nsp.bicep' = {
   name: 'nsp-${uniqueString(resourceGroup().name)}'
+  params: {
+    nspName: svcNSPName
+    location: location
+  }
+}
+
+module svcNSPProfile '../modules/network/nsp-profile.bicep' = {
+  name: 'profile-${uniqueString(resourceGroup().name)}'
   params: {
     accessMode: svcNSPAccessMode
     nspName: svcNSPName
+    profileName: svcNSPName
     location: location
     associatedResources: [
```

Comment:
please add a TODO for service KV, postgres DBs and EV2 service tags

Reply:
Postgres DBs are not supported :(